src/database/migrations/2024_04_24_060621_create_user_profiles_table.php

```diff
@@ -6,24 +6,23 @@
 
 return new class extends Migration
 {
-    /**
+   /**
      * Run the migrations.
      */
     public function up(): void
     {
         Schema::create('profiles', function (Blueprint $table) {
             $table->id();
-            $table->unsignedBigInteger('user_id');
+            $table->unsignedBigInteger('user_id')->index();
```

Comment:
It's not advisable to edit the migration file that was already been migrated. If you want to add, remove, rename a column, do so by adding another migration file.

ref:
https://laracoding.com/how-to-change-a-table-column-type-using-laravel-migrations/